### PR TITLE
Add Library Classes

### PR DESCRIPTION
**Pull request by bimbolabuari**
## OOP School Library 🙂 

 > This PR adds a Microverse project on creating classes and objects in Ruby, Implemented encapsulation and inheritance with Ruby, and run the program using the command line.

### CODE REVIEW

File: person.ru [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age
```
Comment (vichuge) [inline person.ru]: - Please, remember that the requirements ask for:
Getters for @id, @name, and @age.
Setters for @name and @age.
In this case, the @id is getter and setter. Please fix this. 🐶

File: person.ru [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  def initialize(age, name: 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
```
Comment (vichuge) [inline person.ru:14]: - The name of this function is `is_of_age?`, please fix it. 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.
- Please remember, I ask for all changes according to the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md) given to you at the beginning. Fix it and you'll be ready for the next project. 👍 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bimbolabuari** [comment]
@vichuge, Thanks for the review 🚀 

The method name `is_of_age? ` is failing the linter check, that is why I used the name `of_age?`

image

I have fixed all other required changes.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**